guide/inheritance-wallet/wallet-creation.md

```diff
@@ -0,0 +1,290 @@
+    - file: wallet-creation/onboarding-03
+      alt:
+      caption:
+    - file: wallet-creation/onboarding-04
+      alt:
+      caption:
+    - file: wallet-creation/onboarding-05
+      alt:
+      caption:
+    - file: wallet-creation/home-empty
+      alt:
+      caption:
+images_wallet-configuration:
+    - file: wallet-creation/home-empty
+      alt:
+      caption:
+    - file: wallet-creation/creation-add-wallet-sheet
+      alt:
+      caption:
+    - file: wallet-creation/creation-wallet-name
+      alt:
+      caption:
+    - file: wallet-creation/creation-wallet-type
+      alt:
+      caption:
+images_recovery-path:
+    - file: wallet-creation/creation-recovery-disabled
+      alt:
+      caption:
+    - file: wallet-creation/creation-recovery-enabled
+      alt:
+images_inheritance-key-set:
+    - file: wallet-creation/creation-inheritance-empty
+      alt:
+      caption:
+    - file: wallet-creation/creation-inheritance-wallet-type
+      alt:
+      caption:
+    - file: wallet-creation/creation-inheritance-unlock-schedule
+      alt:
+      caption:
+    - file: wallet-creation/creation-inheritance-done
+      alt:
+images_primary-keys-bob:
+    - file: wallet-creation/creation-signing-keys-empty
+      alt:
+      caption:
+    - file: wallet-creation/creation-add-key-sheet
+      alt:
+      caption:
+    - file: wallet-creation/creation-add-key-import-options
+      alt:
+      caption:
+    - file: wallet-creation/creation-add-key-import-flow
+      alt:
+      caption:
+    - file: wallet-creation/creation-signing-keys-2keys
+      alt:
+      caption:
+images_primary-keys-alice:
+    - file: wallet-creation/creation-signing-keys-2keys
+      alt:
+      caption:
+    - file: wallet-creation/creation-request-key-sheet
+      alt:
+      caption:
+    - file: wallet-creation/creation-request-key-name
+      alt:
+      caption:
+    - file: wallet-creation/creation-request-key-share
+      alt:
+      caption:
+    - file: wallet-creation/alice-add-wallet-sheet
+      alt:
+      caption:
+    - file: wallet-creation/alice-scan-request
+      alt:
+      caption:
+    - file: wallet-creation/alice-add-wallet-overview
+      alt:
+      caption:
+    - file: wallet-creation/alice-add-key-import-options
+      alt:
+      caption:
+    - file: wallet-creation/alice-add-key-import-flow
+      alt:
+      caption:
+    - file: wallet-creation/alice-add-done
+      alt:
+      caption:
+images_primary-keys-approve:
+    - file: wallet-creation/creation-key-added-push-notification
+      alt:
+      caption:
+    - file: wallet-creation/creation-key-added-overview
+      alt:
+      caption:
+    - file: wallet-creation/creation-key-bob-review
+      alt:
+      caption:
+    - file: wallet-creation/creation-key-added-reviewed
+      alt:
+      caption:
+    - file: wallet-creation/creation-home-reminders
+      alt:
+      caption:
+images_add-inheritance-keys: 
+    - file: wallet-creation/creation-request-key-share
```

Comment:
Think you mean to use Christina's key request QR image here.

```suggestion
    - file: wallet-creation/creation-request-key-share-1
```

Reply:
I meant this change as a fix to the mistake in the image below (if it is even a mistake). Do confirm this fixes it on your side, I did not test this on local.

<img width="963" alt="dscdfg" src="https://github.com/BitcoinDesign/Guide/assets/114083781/b3780fb6-7702-4900-9ed6-2338ec5dee7f">